mllib/src/test/java/org/apache/spark/mllib/clustering/JavaLDASuite.java

```diff
@@ -110,7 +111,16 @@ public void distributedLDAModel() {
 
     // Check: topic distributions
     JavaPairRDD<Long, Vector> topicDistributions = model.javaTopicDistributions();
-    assertEquals(topicDistributions.count(), corpus.count());
+    // SPARK-5562. since the topicDistribution returns the distribution of the non empty docs
+    // over topics. Compare it against nonEmptyCorpus instead of corpus
+    JavaPairRDD<Long, Vector> nonEmptyCorpus = corpus.filter(
+            new Function<Tuple2<Long, Vector>, Boolean>() {
```

Comment:
Please fix indentation: [https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide]

Actually, I don't think we need to test correctness here since it's tested in the Java unit test